client/components/notes-formatted-block/index.jsx

```diff
@@ -39,7 +40,11 @@ export const FormattedBlock = ( { content = {} } ) => {
 			const url = startsWith( content.url, 'https://wordpress.com/' )
 				? content.url.substr( 21 )
 				: content.url;
-			return <a href={ url }>{ descent }</a>;
+			return (
+				<a href={ url } onClick={ onClick } data-activity={ activity }>
```

Comment:
Can we also add something like `intent` and `section` to this as well as the `site_id` to the possible data points that might be available.